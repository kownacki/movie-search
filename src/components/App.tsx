import * as React from 'react';
import styled from 'styled-components';
import '../index.css';
import Search from './Search';

interface AppProps {
  className?: string,
}

export const App = ({className}: AppProps): React.ReactElement => {
  return (
    <div className={className}>
      <h1>Movie Search</h1>
      <Search />
    </div>
  );
};

export default styled(App)`
  margin: auto;
  max-width: 700px;

  h1 {
    font-weight: normal;
    font-size: 30px;
    text-align: center;
  }
`;
