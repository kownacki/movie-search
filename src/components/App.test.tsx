import React from 'react';
import {shallow} from 'enzyme';
import App from './App';

describe('App', () => {
  it('renders div with Search', () => {
    const classNameStub = 'class-name-stub';
    const app = shallow(<App className={classNameStub} />);
    expect(app).toMatchSnapshot();
  });
});
