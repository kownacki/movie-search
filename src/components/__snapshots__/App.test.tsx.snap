// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders div with Search 1`] = `ShallowWrapper {}`;
